mirascope/cli/commands.py

```diff
@@ -197,19 +197,23 @@ def init(args) -> None:
 
     Args:
         args: The command line arguments for the `init` command, containing:
-            - `directory_name`: The name of the directory to create.
+            - `--mirascope_location`: The root mirascope directory to create.
+            - `--prompts_location`: The user's prompts directory.
     """
     destination_dir = Path.cwd()
-    directory_name = args.directory_name
-    versions_directory = os.path.join(directory_name, "versions")
+    mirascope_location = args.mirascope_location or "mirascope"
```

Comment:
should these be constants?

Reply:
they could be, but since its only used in this file and function i opted to not. i could set the default in cli.py instead

Reply:
thinking again, do we need this `or` at all? since we're defaulting `args.mirascope_location` shouldn't this always have a value (with the default "mirascope")?